src/app/wishlists/selectors.ts

```diff
@@ -29,9 +29,6 @@ export const wishListFunctionSelector = createSelector(
     // Cache of inventory item id to roll. For this to work, make sure vendor/collections rolls have unique ids.
     const cache = new Map<string, InventoryWishListRoll | undefined>();
     return (item: DimItem) => {
-      if (item.id === '0') {
-        return undefined;
-      }
```

Comment:
getInventoryWishListRoll handles the case of id==0 (which is on the next line of code)

Generally speaking a null in this implementation is a Bad Thing since all rolls should have some sort of info (even if it is "not found")

Reply:
Can you document that?